packages/carequality-sdk/README.md

```diff
@@ -0,0 +1,38 @@
+# `carequality-sdk`
+
+SDK to interact with the Carequality directory - by Metriport Inc.
+Built based on the Carequality documentation - https://carequality.org/resources/
```

Comment:
let's be more explicit - the dev now has to look through the CQ dir links to find the actual implementation guide. perhaps:

"SDK to interact with the [Carequality STU3 directory](https://carequality.org/healthcare-directory/index.html) - by Metriport Inc."